src/plugin.js

```diff
@@ -41,24 +48,45 @@ function getUserComponentName(components, name) {
 
   if (typeof components === "object") {
     // { components: ['CldImage'] }
+    // and
+    // { components: [CldImage] }
     if (Array.isArray(components)) {
-      return components.indexOf(name) >= 0 ? name : null;
+      const entry = find(
```

Comment:
why not just use native `array.prototype.find` instead ?

Reply:
https://developer.mozilla.org/pl/docs/Web/JavaScript/Referencje/Obiekty/Array/find
not available on ie